src/components/drawing/index.js

```diff
@@ -100,9 +100,11 @@ drawing.hideOutsideRangePoint = function(d, sel, xa, ya, xcalendar, ycalendar) {
     );
 };
 
-drawing.hideOutsideRangePoints = function(traceGroups, subplot) {
+drawing.hideOutsideRangePoints = function(traceGroups, subplot, selector) {
     if(!subplot._hasClipOnAxisFalse) return;
 
+    selector = selector || '.point,textpoint';
```

Comment:
`'.point,.textpoint'` (you lost a `.`)

Reply:
do we not have an image test that catches this?

Reply:
No. That thing only gets called during pan and zoom. Nice catch. I'll add a pan test to :lock: this down.

Reply:
Fixed and :lock: in -> https://github.com/plotly/plotly.js/pull/2378/commits/b9513a22396454123649581c38903e802bceefa3